package.json

```diff
@@ -17,7 +17,7 @@
     "url": "git+ssh://git@github.com/open-zaak/open-zaak.git"
   },
   "author": "Maykin Media",
-  "license": "EUPL 1.2",
+  "license": "EUPL-1.2",
```

Comment:
Ah, good catch: this now matches SPDX liicense identifier.

https://spdx.org/licenses/